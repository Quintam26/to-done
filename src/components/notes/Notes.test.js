import React from 'react';
import { shallow } from 'enzyme';
import Notes from './Notes';
import toJSON from 'enzyme-to-json';

describe('Notes test', () => {
  
  it('renders add if no note prop', () => {
  
    const notes = [
      { name: 'Nissan', type: 'Car Bio', key: 'abc' },
      { name: 'Top Gear', type: 'Show', key: 'def' },
      { name: 'KOF', type: 'Video Game', key: 'ghi' },
    ];
    const wrapper = shallow(<Notes 
      notes={notes}
      onUpdate={jest.fn()}
      onRemove={jest.fn()}
    />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});