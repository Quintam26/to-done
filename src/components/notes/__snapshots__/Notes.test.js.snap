// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes test renders add if no note prop 1`] = `
<ul>
  <Note
    key="abc"
    note={
      Object {
        "key": "abc",
        "name": "Nissan",
        "type": "Car Bio",
      }
    }
    onRemove={[MockFunction]}
    onUpdate={[MockFunction]}
  />
  <Note
    key="def"
    note={
      Object {
        "key": "def",
        "name": "Top Gear",
        "type": "Show",
      }
    }
    onRemove={[MockFunction]}
    onUpdate={[MockFunction]}
  />
  <Note
    key="ghi"
    note={
      Object {
        "key": "ghi",
        "name": "KOF",
        "type": "Video Game",
      }
    }
    onRemove={[MockFunction]}
    onUpdate={[MockFunction]}
  />
</ul>
`;
